specification/vi/resource-manager/readme.md

```diff
@@ -52,12 +52,10 @@ tag: package-2021-11-10-preview
 These settings apply only when `--tag=2021-11-10-preview` is specified on the command line.
 
 ```yaml $(tag) == 'package-2021-11-10-preview'
-version: 2021-11-10-preview
+version_with_hyphens: 2021-11-10-preview
 version-with-underscores: 2021_11_10_preview
```

Comment:
Does these have effect on SDK?

Reply:
Yes, this parameter collides in its name with the one that specifies the version of autorest core. We have to change this otherwise autorest command will fail immediately complaining that we do not have a version of autorest core `2021-11-10-preview`